contracts/account/signers/SignerP256.sol

```diff
@@ -3,12 +3,14 @@
 pragma solidity ^0.8.20;
 
 import {P256} from "../../utils/cryptography/P256.sol";
-import {Clones} from "../../proxy/Clones.sol";
 import {EIP712ReadableSigner} from "./EIP712ReadableSigner.sol";
 
 abstract contract SignerP256 is EIP712ReadableSigner {
+    bytes32 private immutable _qx;
+    bytes32 private immutable _qy;
+
     function signer() public view virtual returns (bytes32 qx, bytes32 qy) {
-        return abi.decode(Clones.fetchCloneArgs(address(this)), (bytes32, bytes32));
+        return (_qx, _qy);
```

Comment:
Any reason for that change ? My understanding is that it requires redeploying an entier instnace for every new account (that include redeploying the P256 library).

IMO clonesWithArgs are way better here.

Reply:
Yes. I still think we should document how to use signer clones since they are usable for upgradeable accounts where the validation is in its own storage (compliant with the storage validation rules). However, the easiest approach is to use a validator module if it's not upgradeable.

Imo the clones with immutable arguments shouldn't be packaged into the library and rather a section in our documentation where we describe what "immutable signers" are useful for.